### PR TITLE
chat baby

### PR DESCRIPTION
**Pull request by JacobAtchley**

### CODE REVIEW

File: Fabric.Web/Controllers/Api/TestController.cs [hunk 1]
```diff
@@ -19,12 +20,12 @@ public async Task<IActionResult> GetOrleansAsync()
             await client.Connect();
 
             Console.WriteLine("Connected");
-            
-            var grain = client.GetGrain<IMyFirstGrain>(Guid.Parse("26440F3A-D615-4DF9-9E55-A2E740B17C9B"));
 
-            var hello = await grain.SayHello();
+            var grain = client.GetGrain<IMyFirstGrain>(Guid.Empty);
```
Comment (nathanrobinson) [inline Fabric.Web/Controllers/Api/TestController.cs:24]: You could have "channels" where the guid is the channel ID...

Comment (JacobAtchley) [reply Fabric.Web/Controllers/Api/TestController.cs:24]: true

Comment (JacobAtchley) [reply Fabric.Web/Controllers/Api/TestController.cs:24]: i think if we we're going to make it more robust we would

Comment (JacobAtchley) [reply Fabric.Web/Controllers/Api/TestController.cs:24]: i just wanted something to where if i could send a message in a console app it'd show up on the site so we can demonstrate that interaction

**Review by nathanrobinson — APPROVED** [review]

**Pull request merged**